apps/src/dance/DanceVisualizationColumn.jsx

```diff
@@ -26,11 +26,13 @@ const SongSelector = Radium(class extends React.Component {
     setSong: PropTypes.func.isRequired,
     selectedSong: PropTypes.string.isRequired,
     songManifest: PropTypes.arrayOf(PropTypes.object).isRequired,
-    hasChannel: PropTypes.bool.isRequired
+    hasChannel: PropTypes.bool.isRequired,
+    userType: PropTypes.string.isRequired
   };
 
   state = {
-    songsData: []
+    songsData: [],
+    filterOff: this.props.userType !== 'student_y'
```

Comment:
This logic turns the filter off when userType is `unknown`.  It seems like we should write this to err on the 'safe' side so we only turn the filter off if the userType is one of a known set of 'older user' types.